install/kubernetes/cilium/templates/cilium-etcd-operator-deployment.yaml

```
@@ -52,18 +60,12 @@ spec:
           value: "revision"
         - name: CILIUM_ETCD_META_ETCD_AUTO_COMPACTION_RETENTION
           value: "25000"
-{{- if contains "/" .Values.image }}
-        image: "{{ .Values.image }}"
-{{- else if .Values.registry }}
-        image: "{{ .Values.registry }}/{{ .Values.image }}:{{ .Values.tag }}"
-{{- else }}
-        image: "{{ .Values.global.registry }}/{{ .Values.image }}:{{ .Values.tag }}"
-{{- end }}
-        imagePullPolicy: {{ .Values.global.pullPolicy }}
+          image: {{ .Values.etcd.image.repository }}:{{ .Values.etcd.image.tag }}
```

Should indentation like below:

```suggestion
        image: {{ .Values.etcd.image.repository }}:{{ .Values.etcd.image.tag }}
        imagePullPolicy: {{ .Values.etcd.image.pullPolicy }}
```